### DISCUSSION TITLE
Container Scanning

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#13812] [2.4] [answered]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Meets minimum requirements

### CPU

24

### RAM

128

### Storage for /

500GB

### Storage for /nsm

6TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello,
is there a support for container scanning ?
for docker engine native on linux ?
for docker desktop ?
10x

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
> is there a support for container scanning ?

`container scanning` could mean a few different things. What exactly are you trying to accomplish?

**Reply by udi-mosh** [reply]
basically, everything i can, like a real OS.
i am not looking for vulnerability checks (although it would be nice), i am looking for scanning network and security, OS and process operation and etc.
there are some other products that can monitor what going in the container through docker engine or Kubernetes and i am looking to achieve that .
another thing i am trying to achieve is monitoring MS WSL2 operations.

**Reply by dougburks** [reply, marked as answer]
Have you checked the Elastic documentation?
https://www.elastic.co/guide/en/fleet/8.14/install-elastic-agents-in-containers.html

**Reply by udi-mosh** [reply]
thank you very much. this can at list help me solve the container part.
